.gitignore

```diff
@@ -43,9 +43,20 @@ local_test/*
 !local_test/index.html
 local_test/js/*
 local_test/css/*
+# folder of projects to test 
+local_test/intesting
 
 # local test code
 localTestCode.txt
 
 # Apple files
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+
+# textmate
+code_textmate
+code_textmate.tmproj
+
+package-lock.json
```

Comment:
@tchaffee ahhh, this may be problematic. Looks like you are running NPM 5, which this project is not. I don't know how this will affect things. I think it can be safely ignored since the original project does not have a package-lock.json file, we shouldn't bring one into the mix now - my best guess on this.